astropy/coordinates/sky_coordinate.py

```diff
@@ -172,6 +173,15 @@ def __init__(self, *args, **kwargs):
             raise ValueError('Cannot create a SkyCoord without data')
 
     @property
+    def info(self):
+        """
+        Mixin column information (attributes)
+        """
+        if not hasattr(self, '_info'):
+            self._info = ColumnInfo()
+        return self._info
```

Comment:
@mhvk That's what `@lazyproperty` is.  No need for that extra complexity.  If you wanted to be able to have different `ColumnInfo` subclasses that could probably live as an attribute on the `Column` class or something.